pages/api/tribe/find_followers.ts

```diff
@@ -0,0 +1,77 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { connectToDatabase } from "../../../lib/mongodb";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const {
+    query: { username },
+  } = req;
+
+  const appHeaders = new Headers();
+  appHeaders.append("Content-Type", "application/x-www-form-urlencoded");
```

Comment:
Are you sure about the Content-Type header? You don't seem to send an urlencoded form

Reply:
I deleted this file because it was @Marchand-Nicolas files.

I don't use it anymore.